lib/createPropType.js

```diff
@@ -11,8 +11,11 @@ function createPropType (type, validator) {
       return getPropTypeError(type, propName, componentName);
     }
   }
-  Object.defineProperty(propType, 'name', { value: type });
-
+  try {
+    Object.defineProperty(propType, 'name', { value: type })
+  } catch (e) {
+    // https://github.com/benwiley4000/react-extra-prop-types/issues/1
+  }
```

Comment:
Total nit, but could we get a semicolon, and a newline after the `catch`? Thanks! 🙂